### DISCUSSION TITLE
Github project cards frequently remain transparently hovering over the rest

### DISCUSSION THREAD

**Discussion by david-moonsift** [community/community#48496] [Projects and Issues]
### Select Topic Area

Bug

### Body

This bug has been around for a while and it seems to be getting worse.  Dragging cards around often leaves an impression of them stuck floating around the dashboard.  I am not sure exactly what causes it other than moving cards around.

![Screenshot from 2023-02-24 15-36-37](https://user-images.githubusercontent.com/39954127/221222778-62766564-b444-4c9d-95b1-4eefca6036ca.png)

I am using the latest google chrome.

Apologies if this already has been reported, but I couldn't find it and it is getting quite frustrating.  Github projects is unfortunately the most buggy product we are using at the moment and it's not very clear where to report bugs (I usually search for "feedback" when I have issues with websites, expecting a feedback button somewhere).

I also find the whole system quite slow to work with as general feedback.  I have reported some bugs about the iterations feature via the github support but I do not hear back.  Hopefully it is still under development and will improve.

**Comment by rileybroughten** [comment]
Hello @moonsift 👋 Are there any other details you can provide around when you see the lingering impressions when dragging items? If you can provide a screenshot or video of when you see this that would be helpful on our end to look into!

Feel free to give the link for the iterations feedback you have given, I can take a look at that as well 👍

**Reply by david-moonsift** [reply]
Thanks @rileybroughten for your reply.  It happens every so often, and I am yet to figure out how to reproduce it consistently.  Next time it happens I will investigate more and send some screenshots.
